src/keygen.rs

```
@@ -2394,7 +2394,7 @@ impl IndividualPublicKey {
     }
 
     /// Deserialise this individual public key from an array of bytes.
-    pub fn from_bytes(bytes: [u8; 36]) -> Result<IndividualPublicKey, Error> {
+    pub fn from_bytes(bytes: &[u8]) -> Result<IndividualPublicKey, Error> {
```

What if a given serialized key is longer than 36 bytes? Should we return an error?

---

I've updated it, thanks for confirming 😁